### PR TITLE
Add glue api sub landing pages

### PR DESCRIPTION
**Pull request by michelleegan11**
## PR Description

Added subsection landing pages to docs.spryker.com/docs/scos/dev/glue-api-guides/
## Checklist
- [ X] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/glue-api-guides/202204.0/managing-products/managing-products.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Managing products
+description: Describes the endpoints used to manage products
+template: concept-topic-template
+---
+
+This section contains the descriptions of the following endpoints used to manage products in Spryker Commerce OS:
```
Comment (andriitserkovnyi) [inline docs/scos/dev/glue-api-guides/202204.0/managing-products/managing-products.md:7]: Do the people reading our docs know they are reading about SCOS?

File: docs/scos/dev/glue-api-guides/202204.0/managing-b2b-account/managing-b2b-account.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Managing B2B account
+description: Describes the endpoints used to manage a B2B account
+template: concept-topic-template
+---
+
+This section contains the descriptions of the following endpoints used to manage B2B accounts in Spryker Commerce OS:  
+
+* [Searching by company users](/docs/scos/dev/glue-api-guides/202204.0/managing-b2b-account/searching-by-company-users.html)
```
Comment (andriitserkovnyi) [inline docs/scos/dev/glue-api-guides/202204.0/managing-b2b-account/managing-b2b-account.md:9]:
```suggestion
* [Searching by company users](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-b2b-account/searching-by-company-users.html)
```

Comment (andriitserkovnyi) [reply docs/scos/dev/glue-api-guides/202204.0/managing-b2b-account/managing-b2b-account.md:9]: This applies to all the links

File: docs/scos/dev/glue-api-guides/202204.0/managing-products/managing-products.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+This section contains the descriptions of the following endpoints used to manage products in Spryker Commerce OS:  
+* Abstract products
+ 
+  * [Retrieving abstract products](/docs/scos/dev/glue-api-guides/202204.0/managing-products/abstract-products/retrieving-abstract-products.html)
+  
+  * [Retrieving image sets of abstract products](/docs/scos/dev/glue-api-guides/202204.0/managing-products/abstract-products/retrieving-image-sets-of-abstract-products.html)
+  
+* Concrete products
```
Comment (andriitserkovnyi) [inline docs/scos/dev/glue-api-guides/202204.0/managing-products/managing-products.md:14]:
```suggestion
* Concrete products:
```

Comment (andriitserkovnyi) [reply docs/scos/dev/glue-api-guides/202204.0/managing-products/managing-products.md:14]: If your idea about these lists does not need a colon, don't apply this :)

Comment (michelleegan11) [reply docs/scos/dev/glue-api-guides/202204.0/managing-products/managing-products.md]: I added the colons : )

File: docs/scos/dev/glue-api-guides/202204.0/managing-products/managing-products.md (hunk 1 above)
Comment (michelleegan11) [reply docs/scos/dev/glue-api-guides/202204.0/managing-products/managing-products.md:7]: I will leave this as is for now, as we discussed

**Review by andriitserkovnyi — APPROVED** [review]

**Pull request merged**